server_admin/topics/login-settings/remember-me.adoc

```diff
@@ -4,14 +4,29 @@
 A logged-in user closing their browser destroys their session, and that user must log in again. You can set {project_name} to keep the user's login session open if that user clicks the _Remember Me_ checkbox upon login. This action turns the login cookie from a session-only cookie to a persistence cookie.
 
 .Procedure
+ifeval::[{project_product}==true]
 . Click *Realm Settings* in the menu.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Realm settings* in the menu.
 . Click the *Login* tab.
-. Toggle the *Remember Me* switch to *ON*.
+endif::[]
+. Toggle the *Remember Me* switch to *On*.
 +
 .Login tab
+ifeval::[{project_product}==true]
```

Comment:
This screen shot is for the new console, but it also appears in the old console documentation.